src/transformers/models/code_llama/tokenization_code_llama_fast.py

```diff
@@ -256,6 +256,13 @@ def add_bos_token(self, value):
         self.update_post_processor()
 
     def set_infilling_processor(self, reset, suffix_first=False, add_special_tokens=True):
+        """
+        Updates the normalizer to make sure the prompt format for `infilling` is respected. If `reset` is set to
+        `True`, the normalizer is reset to the normal behaviour. The infilling format is the following: if suffix_first
```

Comment:
Sorry to be a stickler, but what is "normal behaviour"?

Reply:
Sorry I'll update you are right 😉 It's the `add_special_tokens` processor that add bos or eos. Will detail this